workflow/scripts/build_scenarios.py

```diff
@@ -45,6 +45,30 @@ def get_shares(df, planning_horizons):
 
     return transport_share, naval_share
 
+def get_transport_growth(df, planning_horizons):
+    # Aviation growth factor - using REMIND-EU v1.1 since DEMO v1 does not include bunkers
+    aviation_model = "REMIND-EU v1.1"
+    aviation = df.loc[aviation_model,"Final Energy|Bunkers|Aviation"]
+    aviation = aviation[planning_horizons]
+    aviation.reset_index(inplace=True)
+    aviation.drop(columns="unit", inplace=True)
+    aviation_growth_factor = aviation / aviation[2020][0]
+
+    # Transport growth factor - using DEMO v1
+    transport_model = "DEMO v1"
+    freight = df.loc[transport_model, "Energy Service|Transportation|Freight|Road"][planning_horizons].reset_index().drop(columns="unit")
+    person = df.loc[transport_model, "Energy Service|Transportation|Passenger|Road"][planning_horizons].reset_index().drop(columns="unit")
+    freight_PJ = df.loc[transport_model, "Final Energy|Transportation|Truck"][planning_horizons].reset_index().drop(columns="unit")
+    person_PJ = df.loc[transport_model, "Final Energy|Transportation|LDV"][planning_horizons].reset_index().drop(columns="unit")
+    
+    transport_growth_factor = pd.DataFrame(columns=planning_horizons)
```

Comment:
use pd.Series instead of pd.DataFrame for easier indexing: `.loc[0,year]` -> `[year]`